packages/shell-api/src/database.ts

```diff
@@ -185,10 +185,14 @@ export default class Database extends ShellApiWithMongoClass {
     cmd: Document,
     options: CommandOperationOptions = {}
   ): Promise<Document> {
-    return this.getSiblingDB('admin')._runCommand(cmd, {
-      ...(await this._baseOptions()),
```

Comment:
Why are we removing the baseOptions here? Not sure the reason.

Reply:
`_runCommand()` already applies it, right?